lib/buy/moonpay/moonpay_provider.dart

```diff
@@ -85,147 +96,92 @@ class MoonPaySellProvider extends BuyProvider {
     }
   }
 
-  Future<Uri> requestMoonPayUrl({
+  Future<Uri> requestSellMoonPayUrl({
     required CryptoCurrency currency,
     required String refundWalletAddress,
     required SettingsStore settingsStore,
   }) async {
-    final customParams = {
+    final params = {
       'theme': themeToMoonPayTheme(settingsStore.currentTheme),
       'language': settingsStore.languageCode,
       'colorCode': settingsStore.currentTheme.type == ThemeType.dark
           ? '#${Palette.blueCraiola.value.toRadixString(16).substring(2, 8)}'
           : '#${Palette.moderateSlateBlue.value.toRadixString(16).substring(2, 8)}',
+      'defaultBaseCurrencyCode': _normalizeCurrency(currency),
```

Comment:
replace all instances of `defaultBaseCurrencyCode` with `defaultCurrencyCode` 
https://dev.moonpay.com/docs/ramps-sdk-buy-params